src/main/resources/templates/db_scripts/schema.json

```diff
@@ -109,6 +109,11 @@
       "run": "after",
       "snippetPath": "migration_scripts/change_progress_total_data_type.sql",
       "fromModuleVersion": "mod-data-export-3.1.0"
+    },
+    {
+      "run": "after",
+      "snippetPath": "migration_scripts/remove_hasSameTagInHoldings.sql",
+      "fromModuleVersion": "mod-data-export-3.1.0"
```

Comment:
we'll release this for bugfix release, so version will be 3.0.1

Reply:
updated